lib/date.ts

```diff
@@ -0,0 +1,23 @@
+const MONTHS = [
+  'Jan',
+  'Feb',
+  'Mar',
+  'Apr',
+  'May',
+  'Jun',
+  'Jul',
+  'Aug',
+  'Sep',
+  'Oct',
+  'Nov',
+  'Dec',
+];
+
+export function formatDatePretty(date: string | Date): string {
```

Comment:
Is it worthy to implement date utilities like these? There are plenty of third party ones that work with locales... right now in catalan, all dates are shown in english

Reply:
This function comes from a copy/paste of an only english project 😆. For me the best solution right now is adding a lang parameter in this utility and support catalan. Said that, if you know a nice tree-shakable date library that could make this formatting (or similar), for me is also good 👌